Resources/Locale/en-US/_NF/accent/streetpunk_accent.ftl

```diff
@@ -0,0 +1,694 @@
+accent-streetpunk-replacement-180 = breath vac
+accent-streetpunk-replacement-181 = dirty
+accent-streetpunk-replacement-182 = dirty
+accent-streetpunk-replacement-183 = makin’
+accent-streetpunk-replacement-184 = choomba
+accent-streetpunk-replacement-185 = nova
+accent-streetpunk-replacement-186 = choomba
+accent-streetpunk-replacement-187 = choombas
+accent-streetpunk-replacement-188 = tin can
+accent-streetpunk-replacement-189 = stims
+accent-streetpunk-replacement-190 = stims
+accent-streetpunk-replacement-191 = meetin’
+accent-streetpunk-replacement-192 = choombas
+accent-streetpunk-replacement-193 = 'dorphs
+accent-streetpunk-replacement-194 = mighta
+accent-streetpunk-replacement-195 = mightna
+accent-streetpunk-replacement-196 = mightna
+accent-streetpunk-replacement-197 = slipup
+accent-streetpunk-replacement-198 = slipups
+accent-streetpunk-replacement-199 = scratch
+accent-streetpunk-replacement-200 = mucha
+accent-streetpunk-replacement-201 = musta
+accent-streetpunk-replacement-202 = mussna
+accent-streetpunk-replacement-203 = mussna
+accent-streetpunk-replacement-204 = handle
+accent-streetpunk-replacement-205 = 'ganic
+accent-streetpunk-replacement-206 = needa
+accent-streetpunk-replacement-207 = screamsheet
+accent-streetpunk-replacement-208 = screamsheet
+accent-streetpunk-replacement-209 = badges
+accent-streetpunk-replacement-210 = handle
+accent-streetpunk-replacement-211 = nothin’
+accent-streetpunk-replacement-212 = badges
+accent-streetpunk-replacement-213 = o’
+accent-streetpunk-replacement-214 = badge
+accent-streetpunk-replacement-215 = ol’
+accent-streetpunk-replacement-216 = 'ganic
+accent-streetpunk-replacement-217 = oughtta
+accent-streetpunk-replacement-218 = outta
+accent-streetpunk-replacement-219 = packin’
+accent-streetpunk-replacement-220 = choom
+accent-streetpunk-replacement-221 = chooms
+accent-streetpunk-replacement-222 = straphanger
+accent-streetpunk-replacement-223 = straphangers
+accent-streetpunk-replacement-224 = plannin’
+accent-streetpunk-replacement-225 = playin’
+accent-streetpunk-replacement-226 = gibson
+accent-streetpunk-replacement-227 = handle
+accent-streetpunk-replacement-228 = gibson
+accent-streetpunk-replacement-229 = horn
+accent-streetpunk-replacement-230 = 'member
+accent-streetpunk-replacement-231 = 'memberin’
+accent-streetpunk-replacement-232 = 'mind
+accent-streetpunk-replacement-233 = 'mindin’
+accent-streetpunk-replacement-234 = media
+accent-streetpunk-replacement-235 = medias
+accent-streetpunk-replacement-236 = tin can
+accent-streetpunk-replacement-237 = tin cans
+accent-streetpunk-replacement-238 = klep
+accent-streetpunk-replacement-239 = klepped
+accent-streetpunk-replacement-240 = klepping
+accent-streetpunk-replacement-241 = ripperdoc
+accent-streetpunk-replacement-242 = kleps
+accent-streetpunk-replacement-243 = rimbo
+accent-streetpunk-replacement-244 = delta
+accent-streetpunk-replacement-245 = delta
+accent-streetpunk-replacement-246 = delta
```

Comment:
Delta part 2?

Examples and replacements:
You going to run away from a fight -> You gonna delta from a fight
He's running away from here -> He's deltain' from here
She pulls out a gun and runs away -> She pulls out a iron (oof) and deltas.

```suggestion
accent-streetpunk-replacement-244 = delta
accent-streetpunk-replacement-245 = deltain'
accent-streetpunk-replacement-246 = deltas
```